client/components/edit/children/ChildList.tsx

```diff
@@ -127,25 +127,26 @@ export const ChildList = ({
     const endNumber = startNumber + pageSize - 1;
     const paginatorLabel =
         children.numFound > 1 ? (
-            <p className={styles.childlist__pagination}>
+            <p class={styles.childlist__pagination}>
```

Comment:
Why the change to `class` here? I thought that was illegal in React and `className` had to be used.

Reply:
Fixed